core/src/main/resources/application.properties

```diff
@@ -33,6 +33,9 @@ spring.jackson.deserialization.accept-float-as-int=false
 spring.jackson.serialization.write-dates-as-timestamps=false
 spring.jackson.serialization.indent_output=true
 
+# User photo upload limit
+spring.http.multipart.max-file-size=128KB
```

Comment:
What about adding a negative test for this?

Reply:
Will add